### ISSUE TITLE
Hash-Mismatch for wofi

### ISSUE THREAD

**Issue by sivizius** [NixOS/nixpkgs#83627]
**Describe the bug**
The hash of wofi is incorrect.

**To Reproduce**
1. `nix-shell -p wofi --pure`
2. error

**Expected behavior**
Installation of wofi

**Error Message**
```
-> % nix-shell -p wofi --pure
these derivations will be built:
  /nix/store/k88xh2sc1wsqgnx5qmlmnrs4wcif4vy9-1cbddafac3806afb203940c029e78ce8390d8f49.drv
  /nix/store/6wx0hrwydapf135bq9gfjccgibs3dy8b-wofi-1.1.2.drv
building '/nix/store/k88xh2sc1wsqgnx5qmlmnrs4wcif4vy9-1cbddafac3806afb203940c029e78ce8390d8f49.drv'...

trying https://paste.sr.ht/blob/1cbddafac3806afb203940c029e78ce8390d8f49
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   477  100   477    0     0    954      0 --:--:-- --:--:-- --:--:--   952
hash mismatch in fixed-output derivation '/nix/store/09zwrqy73k1dfwpprmp9w78hx45pq5wj-1cbddafac3806afb203940c029e78ce8390d8f49':
  wanted: sha256:18960y9ajilrwwl6mjnrh6wj0sm4ivczmacck36p2dj9xd0n8vkh
  got:    sha256:1n4jpmh66p7asjhj0z2s94ny91lmaq4hhh2356nj406vlqr15vbb
cannot build derivation '/nix/store/6wx0hrwydapf135bq9gfjccgibs3dy8b-wofi-1.1.2.drv': 1 dependencies couldn't be built
error: build of '/nix/store/6wx0hrwydapf135bq9gfjccgibs3dy8b-wofi-1.1.2.drv' failed

-> % nix-hash --type sha256 --flat --base32 <(wget -q https://paste.sr.ht/blob/1cbddafac3806afb203940c029e78ce8390d8f49 -O -)
18960y9ajilrwwl6mjnrh6wj0sm4ivczmacck36p2dj9xd0n8vkh
```

**Metadata**
```
-> % nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.5.9, NixOS, 20.09pre217537.d96bd3394b7 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.3`
 - channels(root): `"nixos-20.09pre218482.3320a06049f"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**See also**
* [package](https://github.com/NixOS/nixpkgs/blob/7e62388ed70ca03b80bc8821f9e41323158da685/pkgs/applications/misc/wofi/default.nix)
* [fetchpatch-url](https://paste.sr.ht/blob/1cbddafac3806afb203940c029e78ce8390d8f49)

**Comment by sivizius** [comment]
The file /nix/store/09zwrqy73k1dfwpprmp9w78hx45pq5wj-1cbddafac3806afb203940c029e78ce8390d8f49 misses the first line (`diff -r 309db3c88c9f src/wofi.c
`) but apart from that is the same.

**Comment by petabyteboy** [comment]
Fixed by #83535.
